ANNOUNCE

```diff
@@ -20,8 +20,7 @@ cclib also provides some calculation methods for interpreting the electronic pro
 
 (For a complete list see http://cclib.github.io/methods.html).
 
-For information on how to use cclib, see
-    http://cclib.github.io/tutorial.html
+For information on how to use cclib, see our documentation at http://cclib.github.io.
```

Comment:
What do you think HTTPS instead? Seems like that's become preferred and already works for us.

Reply:
Done, here and in README.